corehq/apps/locations/tests/test_location_fixtures.py

```diff
@@ -330,6 +338,41 @@ def test_include_only_location_types_hierarchical(self):
             ['Massachusetts', 'Suffolk', 'Middlesex']
         )
 
+    def test_get_location_fixture_queryset_with_case_sync_restriction_default_1(self):
+        # Test default - case_sync_restriction on w/o any restrict_cases_to
+        middlesex = self.locations['Middlesex']
+        self.location_types['county'].expand_to = self.location_types['county']  # This should have no effect
+        self.location_types['county'].expand_from = self.location_types['state']  # Also should have no effect
+        self.location_types['county'].save()
+        self.assert_fixture_queryset_equals_locations(
+            ['Massachusetts', 'Middlesex', 'Cambridge', 'Somerville'],
+            [middlesex.id],
+            True
+        )
+
+    def test_get_location_fixture_queryset_with_case_sync_restriction_default_2(self):
+        middlesex = self.locations['Middlesex']
+        self.location_types['county']._expand_from_root = True
+        self.location_types['county'].include_only.set([self.location_types['state']])
+        self.location_types['county'].save()
+        self.assert_fixture_queryset_equals_locations(
+            ['Massachusetts', 'Middlesex', 'Cambridge', 'Somerville'],
+            [middlesex.id],
+            True
+        )
```

Comment:
I don't love this abstraction - feels like you need to look at the definition of `assert_fixture_queryset_equals_locations` to understand what's happening here.  What do you think about something like this?
```suggestion
        self.assert_queryset_equals(
            get_location_fixture_queryset(self.domain, [middlesex.id], case_sync_restriction=True),
            ['Massachusetts', 'Middlesex', 'Cambridge', 'Somerville'],
        )
```